sims/benchmark-simuPOP.py

```diff
@@ -0,0 +1,330 @@
+        help="number of bp in the chromosome",default=1e4)
+parser.add_argument("--theta","-ϴ", type=int, dest="theta",
+        help="theta = 4 N u/site (sites)",default=100)
+parser.add_argument("--rho","-ρ", type=int, dest="rho",
+        help="theta = 4 N r/site (sites)",default=100)
+parser.add_argument("--nsam","-k", type=int, dest="nsamples",
+        help="number of *diploid* samples, total",)
+parser.add_argument("--pdel","-p", type=float, dest="pdel",
+        help="Ratio of deleterious mutations to neutral mutations",)
+parser.add_argument("--record-neutral", type=bool, dest="record_neutral",
+                    default=False)
+
+parser.add_argument("--gamma_shape","-a", type=float, dest="gamma_shape",
+        help="shape parameter in gamma distributed selection coefficients",
+                    default=1.0)
+parser.add_argument("--gamma_scale","-b", type=float, dest="gamma_scale",
+        help="scale parameter in gamma distributed selection coefficients",
+                    default=5.00)
+parser.add_argument("--gc", "-G", dest="simplify_interval", type=int,
+        help="Interval between simplify steps.", default=500)
+parser.add_argument("--logfile","-g", type=str, dest="logfile",
+        help="name of log file (or '-' for stdout)",default="-")
+parser.add_argument("--seed", "-d", dest="seed", type=int, help="random seed")
+
+# optional args
+parser.add_argument("--recomb_rate","-r", type=float, dest="recomb_rate",
+        help="recombination rate",default=1e-7) # was 2.5e-8 but pin to mut rate
+parser.add_argument("--mut_rate","-U", type=float, dest="mut_rate",
+        help="mutation rate of neutral alleles", default=1e-7)
+parser.add_argument("--treefile","-t", type=str, dest="treefile",
+        help="name of output file for trees (default: not output)",default=None)
+parser.add_argument("--generations","-T", type=int, dest="generations",
+        help="number of generations to run for (default 20N)", default=None)
+
+args = parser.parse_args()
+
+if args.generations is None:
+    args.generations = args.popsize * 20
+
+import simuPOP as sim
+
+sim.setRNG(seed=args.seed)
+random.seed(args.seed)
+
+logfile = fileopt(args.logfile, "w")
+
+logfile.write("Options:\n")
+logfile.write(str(args)+"\n")
+logfile.write(time.strftime('%X %x %Z')+"\n")
+logfile.write("----------\n")
+logfile.flush()
+
+# define defaults for benchmark
+npops = 1
+rloci = args.rho / (args.popsize * 4 * args.recomb_rate)
+uloci = args.theta / (args.popsize * 4 * args.mut_rate)
+assert rloci == uloci
+args.nloci = int(rloci)
+args.sel_mut_rate = args.mut_rate * args.pdel
+
+
+# increase spacing between loci as we go along the chromosome
+rel_positions=[0.0 for k in range(args.nloci-1)]
+for k in range(1,args.nloci-1):
+    rel_positions[k] = rel_positions[k-1] + random.expovariate(1)*(k**2)
+pos_fac=args.length/(rel_positions[-1] + random.expovariate(1)*(k**2))
+locus_position=[x*pos_fac for x in rel_positions] + [args.length]
```

Comment:
Get rid of this and just use uniform spacing

Reply:
Look at simuPOP docs for a simple default uniform spacing

Reply:
Change this to agree with defaults (measure length in basepair)

```
lociPos()
return the positions of all loci, specified by the lociPos prameter of the Population function. The default positions are 1, 2, 3, 4, ... on each chromosome.
```

but be explicit